app/views/admin/videos/new.html.haml

```diff
@@ -0,0 +1,23 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        = bootstrap_form_for [:admin, @video], layout: :horizontal, label_col: "col-sm-3", control_col: "col-sm-3" do |f|
+          %ul.nav.nav-tabs
+            %li
+              %a(href="/ui/admin_views_payments") Recent Payments
+            %li.active
+              %a(href="") Add a New Video
+          %br
+          %fieldset
+            = f.text_field :title, label: "Title"
+            = f.select :category_id, options_for_select(Category.all.map {|category| [category.name, category.id]}), control_col: "col-sm-6"
+            = f.text_area :description, rows: 8, label_class: "col-sm-3", control_col: "col-sm-6"
+            = f.file_field :large_cover, control_col: "col-sm-6", label: "Large Cover", class: "btn btn-file"
+            = f.file_field :small_cover, control_col: "col-sm-6", label: "Small Cover", class: "btn btn-file"
+            = f.text_field :video_url, label: "Video URL"
```

Comment:
For better HTML semantics, you could use url_field helper. http://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-url_field